ark/docs/components/apiData.ts

```diff
@@ -585,7 +585,16 @@ export const apiDocsByGroup: ApiDocsByGroup = {
                 [
                     {
                         "kind": "noteStart",
-                        "value": "⚠️ a union that could apply different morphs to the same data is a ParseError ([docs](https://arktype.io/docs/expressions/union-morphs))"
+                        "value": "⚠️ a union that could apply different morphs to the same data is a ParseError ("
```

Comment:
This file is autogenerated so changing it won't work.

You'll need to find the matching jsdoc in `ark/type` and update it there.